pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -155,6 +155,15 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if (kc.ShutdownGracePeriod.Duration > 0 || kc.ShutdownGracePeriodCriticalPods.Duration > 0) && !localFeatureGate.Enabled(features.GracefulNodeShutdown) {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: Specifying ShutdownGracePeriod or ShutdownGracePeriodCriticalPods requires feature gate GracefulNodeShutdown"))
 	}
+	if localFeatureGate.Enabled(features.PodPriorityBasedGracefulShutdown) {
+		if len(kc.PodPriorityShutdownGracePeriods) != 0 && (kc.ShutdownGracePeriod.Duration > 0 || kc.ShutdownGracePeriodCriticalPods.Duration > 0) {
+			allErrors = append(allErrors, fmt.Errorf("invalid configuration: Cannot specifying both PodPriorityShutdownGracePeriods and ShutdownGracePeriod at the same time"))
```

Comment:
minor nit: Cannot specify

Reply:
i had not caught onto this earlier when reviewing but this appears to be the mechanism that prevents negative values from happening in migrate config.  apologies.